src/Command/AuthCommand.php

```diff
@@ -14,20 +16,54 @@
 use Symfony\Component\Validator\Constraints\Regex;
 use Symfony\Component\Validator\Exception\ValidatorException;
 use Symfony\Component\Validator\Validation;
+use Webmozart\KeyValueStore\JsonFileStore;
 
 /**
  * Class CreateProjectCommand.
- *
- * @package Grasmash\YamlCli\Command
  */
-class AuthCommand extends CommandBase {
+class AuthCommand extends Command {
+
+  /**
+   * The default command name.
+   *
+   * @var string
+   */
+  protected static $defaultName = 'auth:login';
```

Comment:
Why do this statically rather than in the configure() method?

Reply:
Doing this statically is required to [support lazy-loading](https://symfony.com/doc/current/console/commands_as_services.html#console-command-service-lazy-loading). I'm not sure why you'd want to use the `configure()` method, maybe that was a best practice in earlier Symfony versions.